blueprints/ember-osf/index.js

```diff
@@ -0,0 +1,30 @@
+/*jshint node:true*/
+var fs = require('fs');
+var path = require('path');
+
+var CONFIG_MARKER = /#+\s+ember-osf\s+settings\s+#+.*?#+(?:.|\s|\n)*/m;
+
+module.exports = {
+    description: '',
+    normalizeEntityName: function() {
+	// h/t: https://github.com/samselikoff/ember-cli-mirage/blob/master/blueprints/ember-cli-mirage/index.js
+        // this prevents an error when the entityName is
+        // not specified (since that doesn't actually matter
+        // to us
+    },
+    afterInstall: function() {
+	var configPath = path.join(this.project.root, 'config', 'local.yml');
+	var tmpConfigPath = path.join(this.project.root, 'config', '_local.yml');
+	try {
+	    debugger;
+	    var contents = fs.readFileSync(configPath).toString();
+	    if (!CONFIG_MARKER.test(contents)) {
+		var tmpConfig = fs.readFileSync(tmpConfigPath).toString();
+		fs.appendFileSync(configPath, tmpConfig);
+	    }
+	    fs.unlinkSync(tmpConfigPath);
+	} catch (e) {
+	    fs.renameSync(tmpConfigPath, configPath);
```

Comment:
What if you changed which one was the temp file? (eg by renaming the existing file before copying over, so that the user is correctly instructed to look at the resulting config file that matters to them) 

As an added advantage, backing up the original file before overwriting it would provide protection if something went wrong. Just overwriting the old file may lead to issues; we may be trying to be *too* smart here.